src/main/app/claims/models/defendantResponse.ts

```diff
@@ -20,7 +20,7 @@ export class DefendantResponse implements Serializable<DefendantResponse> {
       this.id = input.id
       this.claimId = input.claimId
       this.response = new DefendantResponseData().deserialize(input.response)
-      this.defendantDetails = new Defendant().deserialize(input.response.defendant)
+      this.defendantDetails = new Defendant().deserialize(input.defendantDetails)
```

Comment:
That looks like breaking change. There is still `uk.gov.hmcts.cmc.claimstore.models.ResponseData#defendant` property in the backend.

ACTION NEEDED.

Reply:
Done